### PR TITLE
compatibility with foreman 1.18 and 1.19

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: app/services/foreman_ipxe/ipxe_message_renderer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ForemanIpxe
+  class IpxeMessageRenderer
+    attr_accessor :message
+
+    def initialize(message:)
```
Comment (ananace) [inline app/services/foreman_ipxe/ipxe_message_renderer.rb]: Could you give this an explicit default value - `message: nil` or just anything really, that colon tickles some kind of nerve on me.

File: .rubocop.yml [hunk 2]
```diff
@@ -1,9 +1,7 @@
----
```
Comment (ananace) [inline .rubocop.yml]: Any particular reason why you're removing the document header?

Most YAML linters will complain mightily about the lack of one

**Review by ananace — CHANGES_REQUESTED** [review]
Just a tiny bit of nitpicking

**Comment by timogoebel** [comment]
@ananace: Addressed your comments.

**Review by ananace — APPROVED** [review]

**Comment by ananace** [comment]
LGTM in that case.

**Pull request merged**